### PR TITLE
virtmanager: 1.4.3 -> 1.5.0, cleanup dependencies

### PR DESCRIPTION
**Pull request by xeji**
remove unneeded dependencies.
closes issue #34043 

###### Things done

- [] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

/cc maintainers:  @qknight @offlinehacker  @fpletz

### CODE REVIEW

File: pkgs/applications/virtualization/virt-manager/default.nix [hunk 1]
```diff
@@ -9,12 +9,12 @@ with stdenv.lib;
...
 python2Packages.buildPythonApplication rec {
   name = "virt-manager-${version}";
-  version = "1.4.3";
+  version = "1.5.0";
   namePrefix = "";
 
   src = fetchurl {
     url = "http://virt-manager.org/download/sources/virt-manager/${name}.tar.gz";
-    sha256 = "093azs8p4p7y4nf5j25xpsvdxww7gky1g0hs8mkcvmpxl2wjd0jj";
+    sha512 = "b375927776b9132fbd9dacd8223b6c94b89c32d6812394ec7e18df7c66f7e6dec853885e85e2b4b4ffd283e8afe0dd2526bafeac4b55511a4a115ef5798f97da";
```
Comment (volth) [inline pkgs/applications/virtualization/virt-manager/default.nix]: do we really need the hashes wider than the display  ?

Comment (xeji) [reply pkgs/applications/virtualization/virt-manager/default.nix]: good point. changing  to sha256.

**Comment by xeji** [comment]
@GrahamcOfBorg  eval

**Comment by jtojnar** [comment]
@GrahamcOfBorg eval

**Comment by joachifm** [comment]
@GrahamcOfBorg build virtmanager

**Comment by xeji** [comment]
The libvirt-glib dependency isn't new and explicitly required in virt-manager source. Don't think we can change that.

**Comment by jtojnar** [comment]
@xeji it is required for example here: https://github.com/virt-manager/virt-manager/blob/1302156265c9b340b48ea2c99b6079c7c92e8d76/virt-manager#L30

**Comment by xeji** [comment]
@jtojnar so should we exclude darwin in virtmanager for the time being?

**Comment by jtojnar** [comment]
@xeji We could just ignore it for now. Though if you want, you could try changing the `platforms` attribute to `unix`, libvirt is supposed to support Darwin (https://libvirt.org/).

**Comment by jtojnar** [comment]
@GrahamcOfBorg eval

**Comment by jtojnar** [comment]
@GrahamcOfBorg build virtmanager

**Comment by xeji** [comment]
@jtojnar this seems to be more complicated on Darwin. Let's reverse the last commit and set `platforms=linux` in virtmanager as well. If anyone needs this on Darwin, they can figure it out.

**Comment by jtojnar** [comment]
@GrahamcOfBorg build virtmanager

**Comment by xeji** [comment]
Build fails on linux apparently caused by: rustc-1.24 build timeout.
@Mic92  could that be related to f61e8d98ff7fbb049ddeea33742bb4b840dafbfa ?

**Comment by Mic92** [comment]
@xeji hopefully resolved soon, when hydra catches up.

**Comment by jtojnar** [comment]
@GrahamcOfBorg build virtmanager

**Pull request merged**